### PR TITLE
Add Install Guide

### PR DESCRIPTION
**Pull request by iranzo**
Add Install guide

### CODE REVIEW

**Comment by iranzo** [comment]
@rlopez133 @DirectedSoul1 @romfreiman 

Can you give it a look to compare about the original document to see if there's something to fix?

Thx!

File: GUIDE.md [hunk 1]
```diff
@@ -0,0 +1,167 @@
+# Assisted Installer Unofficial Install Guide
+
+> DISCLAIMER: These are just steps we have used to do the deployments. They are by no means "official" ; it is just our team's experience in how we've been able to deploy and we are providing a reproducible method. Input on making this better is welcomed and encouraged!
+
```
Comment (rlopez133) [inline GUIDE.md:4]: We should remove this disclaimer.

File: GUIDE.md [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+        [$USER@assisted_installer ~]# for i in <master0-idrac-ip> <master1-idrac-ip> <master2-idrac-ip>; do podman run --net=host idracbootfromiso -r $i -u <idrac-user> -p "<idrac-pw>" -i http://       <host-ip>:8080/live.iso; done
+
+    > NOTE1: Ensure to include the proper host **iDRAC IPs**, **iDRAC user** and **iDRAC password**. This for loop assumes iDRAC user and iDRAC password are the same, if different adjust the shell command accordingly.
+
+
+    > NOTE2: Also make sure to **remove/eject the older ISO** which were mounted using `racadm` command as shown below
+
+        sshpass -p '*****' ssh root@<iDRAC-IP> racadm remoteimage -d
+
+1.  The `live.iso` should reboot the nodes and boot them into a Fedora Live image. Once it has done this, shortly you will notice the nodes becoming discoverable for your cluster via the `http://<host-ip>:6008/clusters/<cluster-id>` dashboard.
```
Comment (rlopez133) [inline GUIDE.md:136]: If we get the Ansible playbook merged, this will all change.

Comment (iranzo) [reply GUIDE.md:136]: it can be amended later :)

Comment (DirectedSoul1) [reply GUIDE.md:134]: yeah, also we can keep this guide and provide steps for ansible in a sub-section? just a suggestion.

File: GUIDE.md [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+
+As `$USER` user with sudo privileges ,
+
+1.  Generate an SSH key
+
+        [$USER@assisted_installer ~]# ssh-keygen -t rsa -f ~/.ssh/id_rsa -P ''
+
+1.  Install git and make on your CentOS assisted installer host if not already present
+
+        [$USER@assisted_installer ~]# dnf install -y make git
```
Comment (tsorya) [inline GUIDE.md:28]: lets add make here

File: GUIDE.md [hunk 4]
```diff
@@ -0,0 +1,165 @@
...
+1.  Create a directory under `~/` labeled `assisted-installer` and `git clone` the `test-infra` git repository
+
+        [$USER@assisted_installer ~]# mkdir ~/assisted-installer
+        [$USER@assisted installer assisted-installer]# cd ~/assisted-installer/
+        [$USER@assisted_installer assisted-installer]# git clone https://github.com/openshift/assisted-test-infra
+        [$USER@assisted_installer assisted-installer]# cd assisted-test-infra
+
+1.  Setup the assisted installer's environment by running the `install_environment.sh` script
+
+        [$USER@assisted_installer assisted-test-infra]# ./scripts/install_environment.sh
```
Comment (tsorya) [inline GUIDE.md:39]: no need. just run "make create_full_environment" and move make to the line where git is installed.

File: GUIDE.md [hunk 5]
```diff
@@ -0,0 +1,165 @@
...
+
+## Deployment
+
+Procedure
+
+As `$USER` user with sudo privileges ,
+
+1.  Generate an SSH key
+
+        [$USER@assisted_installer ~]# ssh-keygen -t rsa -f ~/.ssh/id_rsa -P ''
```
Comment (tsorya) [inline GUIDE.md:24]: in test-infra u have have folder with ssh keys. so u can just use them
ssh_key/key

File: GUIDE.md [hunk 6]
```diff
@@ -0,0 +1,165 @@
...
+        [$USER@assisted_installer assisted-test-infra]# make run
+        .
+        .
+        .
+
+
+        deploy_ui.sh: OCP METAL UI can be reached at http://<host-ip>:6008
+        deploy_ui.sh: Done
+
+1. Once the UI has finished deploying, on a browser, access it via your host's IP and use port 6008, URL will be something like `http://<host-ip>:6008`
```
Comment (tsorya) [inline GUIDE.md:60]: all this can be done by running "make download_iso"

File: GUIDE.md (hunk 3 above)
Comment (iranzo) [reply GUIDE.md:28]: `make` is already there, something I'm missing?

File: GUIDE.md (hunk 4 above)
Comment (iranzo) [reply GUIDE.md:39]: so after the install make, git, just run 'make create_full_environment' right?

File: GUIDE.md (hunk 3 above)
Comment (tsorya) [reply GUIDE.md:28]: my fault

File: GUIDE.md (hunk 4 above)
Comment (tsorya) [reply GUIDE.md:39]: right

File: GUIDE.md (hunk 6 above)
Comment (rlopez133) [reply GUIDE.md:60]: @tsorya while this is true, I think the real question is what do you want the experience to be like? Would you prefer people use the UI or use the make download_iso?

File: GUIDE.md (hunk 5 above)
Comment (rlopez133) [reply GUIDE.md:24]: @tsorya @iranzo However,  if you have a folder with ssh keys, there is an assumption that ssh keys exist. Likely a better approach here would be to say generate an ssh key if not available.

File: GUIDE.md (hunk 6 above)
Comment (tsorya) [reply GUIDE.md:60]: u are right but i think we will have 2 types of users so we can at least describe the second option?

**Comment by tsorya** [comment]
looks great. thanks

**Pull request merged**